### DISCUSSION TITLE
GitHub Copilot could not connect to server: ` invalid ELF header`

### DISCUSSION THREAD

**Discussion by scarf005** [community/community#41105] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

everytime I start up VSCode, I get this same error: and I have to press 'retry' button.
```
[ERROR] [default] [2022-12-08T14:38:14.861Z] GitHub Copilot could not connect to server. Extension activation failed: "/home/scarf/.vscode/extensions/sporiley.symbol-masks-0.0.4/node_modules/oniguruma/build/Release/onig_scanner.node: invalid ELF header"
```

**Comment by Akash1134** [comment, marked as answer]
Hi @scarf005 , We are sorry that you are not able to use Copilot. Could you please try restarting VS Code and see if it helps? If not so, sign out of your GitHub account from your VS Code then disable and re-enable the Copilot extension. Let us know how it goes!
